admin-ui/plugins/auth-server/helper/utils.js

```diff
@@ -0,0 +1,68 @@
+// LDAP utility functions
+export function formatLdapServers(servers) {
+  if (Array.isArray(servers)) {
+    return servers.join(', ')
+  }
+  return servers
+}
+
+export function formatBaseDNs(baseDNs) {
+  if (Array.isArray(baseDNs)) {
+    return baseDNs.join(', ')
+  }
+  return baseDNs
+}
+
+export const lDapFormInitialState = (isEdit, item) =>
+  isEdit && item
+    ? item
+    : {
+        configId: '',
+        bindDN: '',
+        bindPassword: '',
+        servers: [''],
+        maxConnections: '',
+        baseDNs: [''],
+        primaryKey: '',
+        localPrimaryKey: '',
+        level: '',
+        defaultAuthnMethod: false,
+        useSSL: false,
+        useAnonymousBind: false,
+        enabled: false,
+        version: '',
+      }
```

Comment:
Using an inconsistent camel-casing convention for a function name is bad because it violates the principle of consistent coding style by deviating from the project's established naming conventions, which harms code readability and makes the codebase harder to maintain.